tests/lambda_endpoints/test_ialirt_ingest.py

```diff
@@ -0,0 +1,55 @@
+"""Test the IAlirt ingest lambda function."""
+
+import os
+
+import boto3
+import pytest
+from moto import mock_dynamodb
+
+from sds_data_manager.lambda_code.IAlirtCode.ialirt_ingest import lambda_handler
+
+
+@pytest.fixture()
+def dynamodb_table():
```

Comment:
Do we need to mock this again here? I think we can put it into a `conftest.py` to share.